capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable no-unused-vars */
+
+/*
+ * Largely inspired by JS code from W3Schools: https://www.w3schools.com/howto/howto_js_autocomplete.asp
+ */
+const SEARCH_INPUT = document.getElementById('search');
+const ITEM_CLASS = 'autocomplete-items';
+const ACTIVE_CLASS = 'autocomplete-active';
+const SUGGESTION_LIST_ID = 'autocomplete-list';
+const UP_KEY = 'ArrowUp';
+const DOWN_KEY = 'ArrowDown';
+const ENTER_KEY = 'Enter';
+let selectedSuggestionPosition = -1;
+
+/**
+ * Adds the 'active' tag to an autocomplete elmt
+ * @param {HTMLDivElement} autocompleteListElmt
+ */
+function addActiveTag(autocompleteListElmt) {
```

Comment:
If possible try to avoid abbreviations. autocompleteListElement would read better.

Reply:
ah thanks! done!